fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -957,12 +1074,17 @@ private RecordType addIndexAndSaveRecordForAutoComplete(@Nonnull FDBRecordContex
 
     private void assertDocumentPartialRecordFromIndexEntry(@Nonnull RecordType recordType, @Nonnull IndexEntry indexEntry,
                                                            @Nonnull String expectedSuggestion, @Nonnull String fieldName) {
-        Descriptors.Descriptor recordDescriptor = recordType.getDescriptor();
-
-        Message message = LuceneIndexQueryPlan.getToPartialRecord(indexEntry.getIndex(), recordType)
-                .toRecord(recordDescriptor, indexEntry);
+        Descriptors.Descriptor recordDescriptor = TestRecordsTextProto.SimpleDocument.getDescriptor();
+        IndexKeyValueToPartialRecord toPartialRecord = LuceneIndexQueryPlan.getToPartialRecord(SIMPLE_TEXT_WITH_AUTO_COMPLETE, recordType, IndexScanType.BY_LUCENE_AUTO_COMPLETE);
```

Comment:
I think this may be a merge skew: it's ignoring the passed in `recordType` and the passed in `indexEntry`'s index and using fixed values for both.

It may be this is why some auto-complete tests are failing in CI.